Streetcode/Streetcode.WebApi/Program.cs

```diff
@@ -30,6 +30,6 @@
 
 app.MapControllers();
 
-await dbTask;
+//await dbTask;
```

Comment:
Why are lines `11` and `33` commented out?

Reply:
already fixed it